edit-post/components/sidebar/block-sidebar/style.scss

```diff
@@ -10,6 +10,10 @@
 		color: $dark-gray-500;
 	}
 
+	> div:last-child {
+		margin-bottom: 0;
```

Comment:
Since the style is applied above on `.components-base-control`, could we just add the style as nested `&:last-child` there? Or even just update the selector to `.components-base-control:not(:last-child)`